test/unit/models/users.test.js

```diff
@@ -488,4 +488,53 @@ describe("users", function () {
       expect(userListResult[0].discordId).to.be.deep.equal(userDataArray[0].discordId);
     });
   });
+
+  describe("fetchUser with disabled roles", function () {
+    afterEach(async function () {
+      await cleanDb();
+    });
+
+    it("should fetch users with modified roles : []", async function () {
+      const superUser = { ...userData()[4], disabled_roles: [] };
+      const userId = await addUser(superUser);
+
+      const userDoc = await users.fetchUser({ userId });
+      expect(userDoc.user.disabled_roles.length).to.be.equal(0);
+      expect(userDoc.user.roles.super_user).to.be.equal(true);
+    });
+
+    it("should fetch users with modified roles : super_user", async function () {
+      const superUser = { ...userData()[4], disabled_roles: ["super_user"] };
+      const userId = await addUser(superUser);
+
+      const userDoc = await users.fetchUser({ userId });
+      expect(userDoc.user.disabled_roles.length).to.be.equal(1);
+      expect(userDoc.user.roles.super_user).to.be.equal(false);
+    });
+
+    it("should fetch users with modified roles : member", async function () {
+      const memberUser = { ...userData()[6], disabled_roles: ["member"] };
+      const userId = await addUser(memberUser);
+
+      const userDoc = await users.fetchUser({ userId });
+      expect(userDoc.user.disabled_roles.length).to.be.equal(1);
+      expect(userDoc.user.roles.member).to.be.equal(false);
+    });
+
+    it("should fetch users with modified roles : super_user & member", async function () {
+      // super_user & member
```

Comment:
Do we need this comment as test message is already declarative

Reply:
sure i'll fix this